.github/workflows/release.yaml

```diff
@@ -0,0 +1,47 @@
+name: "Publish module"
+
+on:
+  workflow_dispatch:
+
+jobs:
+  create-github-release:
+    name: Deploy GitHub Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.ref }}
+          clean: true
+          fetch-depth: 0
+      - name: Get Version
+        id: gv
+        run: |
+          echo "::set-output name=ver::$(jq --raw-output .version metadata.json)"
+      - name: Create Release
+        uses: actions/create-release@v1
+        id: create_release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: "v${{ steps.gv.outputs.ver }}"
```

Comment:
```suggestion
          tag_name: "${{ steps.gv.outputs.ver }}"
```

Reply:
It looks like this applies to all the modules I think 😬

Reply:
Other than that LGTM!

Reply:
💀 
Looks like the pkcs7 branch I copied this from didn't have this change.

Reply:
K, all PRs should be updated.